Automattic/woocommerce-payments
# Fixed payment method name on success page for saved cards #3027

Fixes #3027

#### Changes proposed in this Pull Request
This PR changes how the name of a payment method is set during checkout. To make sure the proper name is set instead of just showing WooCommerce Payments on the success page

#### Testing instructions
- As a merchant activate upe checkout
- As a shopper go to My account > Payment methods > Add Payment Method and add the credit card ```4242 4242 4242 4242```
- Buy any product and checkout using the saved product
- The payment method should be set to "Vista credit card" on the success page

-------------------


**Post merge**
